apps/02RockPaperScissors.js

```diff
@@ -7,10 +8,47 @@ prompt.start();
 
 function rockPaperScissors(hand1, hand2) {
 
-    // Write code here
 
-}
+  hand1 = hand1.toLowerCase();
+  hand2 = hand2.toLowerCase();
+
+  if (hand1 === hand2){
+    return "It's a tie!";
+  }
 
+  if (hand1 ==='rock'){
+    if (hand2 === 'scissors'){
+      return "Hand one wins!";
+      }
+  }
+  if (hand1 ==='rock'){
+    if (hand2 ==='paper'){
+      return "Hand two wins!";
+     }
+  }
+  if (hand1 === 'paper'){
+    if (hand2 === 'rock'){
+      return "Hand one wins!";
+      }
+  }
+  if (hand1 ==='paper'){
+    if (hand2 ==='scissors'){
+      return "Hand two wins!";
+      }
+  }
+
+  if (hand1 ==='scissors'){
+      if (hand2 === 'paper'){
+        return "Hand one wins!";
+        }
+      }
+
+  if (hand1 ==='scissors'){
+    if (hand2 ==='paper'){
+      return "Hand two wins!";
+      }
+    }
```

Comment:
@kelleymccauley Your Rock, Paper, Scissors tests pass, but I think this if statement is incorrect here.

If **hand1** is **scissors**, and **hand2** is **paper**, does **hand2** win?